src/commands.ts

```diff
@@ -160,7 +162,18 @@ export function registerCommands(context: vscode.ExtensionContext, prManager: Pu
 		if (isPartial) {
 			vscode.window.showInformationMessage('Your local repository is not up to date so only partial content is being displayed');
 		}
-		vscode.commands.executeCommand('vscode.diff', parentFilePath, filePath, fileName, opts);
+
+		// if this is an image, encode it as a base64 data URI
+		const imageDataURI = await asImageDataURI(parentFilePath, prManager.repository);
+		const originalURI = imageDataURI || parentFilePath;
+
+		if (fileChangeNode.status === GitChangeType.ADD) {
```

Comment:
I'm not sure if we should do this or not - on the one hand it adds a bunch of screen real estate when viewing these files, but it also makes it less obvious at first glance if they are an add or delete. My preference is for the old way

Reply:
Moved to using a transparent pixel URI, which is what the builtin diff view does.